jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibPluginTest.java

```diff
@@ -137,7 +137,7 @@ public void testCheckJibVersionInvoked() {
     // Gradle tests aren't run from a jar and so don't have an identifiable plugin version
     assertThat(exception)
         .hasMessageThat()
-        .isEqualTo("Failed to apply plugin [id 'com.google.cloud.tools.jib']");
+        .isEqualTo("Failed to apply plugin 'com.google.cloud.tools.jib'.");
```

Comment:
this error message changed as a result of the gradle wrapper upgrade from 6.0.1 to 6.6.1